### DISCUSSION TITLE
How to set preset in modelspecs for new agents feature ?

### DISCUSSION THREAD

**Discussion by RAPHCVR** [danny-avila/LibreChat#4449] [Help Wanted]
I would like to be able to set a preset for agents, in modelspecs, as for assistants, in order for them to have unique and default settings, with custom instructions (or using the one already set), custom name and logo, etc.

How to proceed ?

Here is the example with azure openai assistant (azureassistant id being the unique id of the assistant created):
![image](https://github.com/user-attachments/assets/9ccdafee-bea8-4ab2-b202-c9bd4e375fb2)

Do not hesitate if there are other ways to proceed.

Thanks in advance !

**Comment by danny-avila** [comment]
The new agents feature is not fully complete yet, this is one of the remaining items.

**Reply by RAPHCVR** [reply]
Thanks for the quick answer !
This new agent system is really impressive btw.
Waiting for this feature.

**Reply by danny-avila** [reply]
Thank you! I'm working every day towards this goal
